tests/io_utils.go

```diff
@@ -69,13 +69,19 @@ func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (stdout stri
 
 func CreateErrorDisk(nodeName string) (address string, device string) {
 	By("Creating error disk")
-	args := []string{"/usr/bin/virt-chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/modprobe", "scsi_debug", "opts=2", "every_nth=4"}
+	return CreateSCSIDisk(nodeName, []string{"opts=2", "every_nth=4"})
+}
+
+// CreateSCSIDisk creates a SCSI disk using the scsi_debug module. This function should be used only to check SCSI disk functionalities and not for creating a filesystem or any data. The disk is stored in ram and it isn't suitable for storing large amount of data.
+func CreateSCSIDisk(nodeName string, opts []string) (address string, device string) {
+	args := []string{"/usr/bin/virt-chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/modprobe", "scsi_debug"}
```

Comment:
`scsi_debug` sounds like something which we could enable in the kubevirtci environments.